### DISCUSSION TITLE
thino引用问题

### DISCUSSION THREAD

**Discussion by jackbobcc** [Quorafind/Obsidian-Thino#698] [Ideas 想法]
如果一条thino被引用，目前只能在点开阅读该条thino时才能显示被引用，建议在主页面中该条thino下也显示引用提示，类似flomo，
![image](https://github.com/user-attachments/assets/9cdaeb85-4c42-42ae-a3a0-fedea7a12a39)

**Comment by saralaaga** [comment]
显示被引可以很好的知道卡片之间关系的方式，希望能够完善这一点。
我甚至一度以为这是bug
